Desktop/analysis/analysisform.cpp

```diff
@@ -224,10 +218,21 @@ void AnalysisForm::_setUp()
 		}
 	}
 
-	std::sort(controls.begin(), controls.end(), 
+	std::sort(controls.begin(), controls.end(),
 		[](JASPControl* a, JASPControl* b) {
 			return a->depends().size() < b->depends().size();
```

Comment:
Would this not go wrong if you have:
- D depends on C
- C depends on B and A
- B depends on A

Then the order is [ A, B, D, C ] right?

Reply:
But it should be [ A, B, C, D] no?

Reply:
And if this works, because the `depends()` adds all dependencies recursively so that "D depends on C and thus on B and A" then it would be good to add that as a comment here.